app/frontend/styles/_filter.scss

```diff
@@ -0,0 +1,31 @@
+.moj-action-bar__filter:after {
+  background: none;
+}
+
+@media (min-width: 48.0625em) {
```

Comment:
@adamsilver is there a way to rename these to `.dfe-...`? Not a huge deal but jars a little.